examples/tools/use-cases/cascading-inputs-tool-showcase/README.md

```diff
@@ -1,5 +1,5 @@
-# Flow with enabled_by_value
-A standard flow using "enabled_by_value" python tool. 
+# Basic flow with package tool using cascading inputs
+A basic flow with package tool that using cascading inputs.
```

Comment:
This is a flow demonstrating the use of a tool with cascading inputs. Cascading inputs are frequently used in situations where the selection in one input field determines what subsequent inputs should be shown. It helps in creating a more efficient, user-friendly, and error-free input process.